Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/Actions/MouseActionsTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Aquality.WinAppDriver.Actions;
+using Aquality.WinAppDriver.Elements.Interfaces;
+using Aquality.WinAppDriver.Tests.Applications.Locators;
+using NUnit.Framework;
+using OpenQA.Selenium.Interactions;
+
+namespace Aquality.WinAppDriver.Tests.Elements.Actions
+{
+    public class MouseActionsTests : Tests.Actions.MouseActionsTests
+    {
+        protected override IMouseActions MouseActions => RightArgumentTextBox.MouseActions;
+        private ITextBox LeftArgumentTextBox => new CalculatorWindow().LeftArgumentTextBox;
+
+        [Test]
+        public void Should_PerformElementSpecificMouseActions()
```

Comment:
Please, create new issue in github to create more specific tests. Because now this test does not show as is really drag and drop works or not

Reply:
It is already exists: #15 
Cover MouseActions with appropriate tests: test that action is performed correctly
https://github.com/aquality-automation/aquality-winappdriver-dotnet/issues/15